.hintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": [
+    "development"
+  ],
```

Comment:
do you need this?

Reply:
nope, might have been some install thing